gslib/tests/test_stat.py

```diff
@@ -141,9 +141,9 @@ def test_stat_one_missing_wildcard(self):
   def test_stat_bucket_wildcard(self):
     bucket_uri = self.CreateBucket()
     self.CreateObject(bucket_uri=bucket_uri, object_name='foo', contents=b'z')
-    stat_string = suri(bucket_uri)[:-1] + '?/foo'
+    stat_string = suri(bucket_uri) + '/?oo'
     self.RunGsUtil(['stat', stat_string])
-    stat_string2 = suri(bucket_uri)[:-1] + '*/foo'
+    stat_string2 = suri(bucket_uri) + '/*foo'
```

Comment:
Along with my comment above, please move the wildcard characters back into the bucket portion of these URIs.